pkg/aws/types.go

```diff
@@ -65,6 +65,8 @@ const (
 	AccessKeyID = "accessKeyID"
 	// SecretAccessKey is a constant for the key in a cloud provider secret and backup secret that holds the AWS secret access key.
 	SecretAccessKey = "secretAccessKey"
+	// SharedCredentialsFile is a constant for the key in cloud provider secret that holds the AWS credentials file.
+	SharedCredentialsFile = "cloudprovider.credentials"
```

Comment:
The secret is already called `cloudprovider`. What is the rationale of duplicating this into this key? Just call it `credentials`?

Reply:
Agreed, I have chosen a poor name. Maybe I can better name it `credentialsFile` since we already have credentials in the secret?

Reply:
Sounds good :)